src/assets/avatar-silhouette.svg

```diff
@@ -0,0 +1,11 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 136 136">
+  <defs/>
+  <mask id="a" width="136" height="136" x="0" y="0" maskUnits="userSpaceOnUse">
+    <circle cx="68" cy="68" r="68" fill="white"/>
+  </mask>
+  <g mask="url(#a)">
```

Comment:
Just the idea. Previously we had some issues with `clip-path` property with SVGs that are over `CoverVideo`. In `SearchOverlay` this SVG can be present. Might be worth checking if it behaves properly with `mask` property as well. What do you think?

Reply:
I move around this SVG and it seems to be working.
![image](https://user-images.githubusercontent.com/51168865/107752718-6d4b5780-6d1f-11eb-949b-43854ce3703b.png)